packages/react-mutation-mapper/src/util/DataFetcherUtils.ts

```diff
@@ -85,7 +85,7 @@ export async function fetchVariantAnnotationsByMutation(
 
 export async function fetchVariantAnnotationsIndexedByGenomicLocation(
     mutations: Partial<Mutation>[],
-    fields: string[] = ['annotation_summary'],
+    fields: any = 'annotation_summary',
```

Comment:
same as above. let's keep this param (and the default value) as is if possible and do `fields as any` where we call the fetch function.